src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -374,7 +379,11 @@ const MetaTraderConfig = (() => {
                     resolve(needsRealMessage());
                 } else if (accounts_info[acc_type].account_type === 'financial') {
                     BinarySocket.send({ get_account_status: 1 }).then(() => {
-                        resolve(!isAuthenticated() ? $messages.find('#msg_authenticate').html() : '');
+                        if (getMTFinancialAccountType(acc_type) === 'advanced' && isAuthenticationPromptNeeded()) {
```

Comment:
can we change this back to `!isAuthenticated`? prompt to authenticate will only show for high risk